ruby_library/entities/library.rb

```diff
@@ -0,0 +1,100 @@
+require_relative '../storage/storage'
+
+class Library
+    include Storage
+
+    def initialize(authors=[], books=[], readers=[], orders=[])
+        @authors = []
```

Comment:
if we have more than 2 args - please use keyword arguments/named arguments
`authors: [], books: [] ...`

Reply:
ok